### PR TITLE
Add base repository files

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Comment by guewen** [comment]
What about translations? I didn't put transifex because I heard there is a new system :)

**Comment by pedrobaeza** [comment]
Well, I'm afraid not yet.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - expect-dev  # provides unbuffer utility
+      - python-lxml # because pip installation is slow
+
+env:
+  global:
+  - VERSION="11.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+
+  matrix:
+  - LINT_CHECK="1"
+  - TESTS="0" ODOO_REPO="odoo/odoo"
```
Comment (pedrobaeza) [inline .travis.yml]: Put TESTS="1" in these 2 lines.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Use this Transifex secure
```
  secure: "jErjXdqkLMHIwGJIWJW0kJiL0MZsPutBHRkdZ0ERIC9q9Td6oExPcaiYuNMwEJsetoA51dnOuTk/pNCFvYZGectDXQgVk5D9vXYiHfwZZT56rTA5IrscrPr8LobFrV7i7JY1+wIK8lOsfEwZvmhV6NSg+vbI9UpP6sI5W8QSF3dfqh56NsaKII9v/q12S/QxkhQdIgqpg6le+yooFFNmLs9M1kqbsoGYOXnsNHx6wGuY/N5l16td3A2eO4ulCZLUu1Djc0PiwI6ZAJue50ZfMj3pW20X/+u9pSBUPBC2Jh77VXFln50A++0OZ27uYGsQJOC2abg7Zze6gdYz3qoIGq13F+wwqKkJ8rsfN9yI6Z41GnRQ6NOg+j/YWUJ6kWY4VDDHeYfLqX7AiGHLTsCcn/FV3WnoMSVj6Rm/nZoBZ3K9nrVaolCd5oFtddBXzvZHqdTKHRzzRyLLq9Lo+FBxoVjwQycHhoWlyFf39gu1cLGLKHaM9nSDjLTNXZTWtcS/kookNxb9QfYXGTEMPeXYlvOjQ6QaKeJ0Qrg1LQgsMb9cHUmGjdu525Nx1XJb5wkz0q21EbF88HlKzxtBKCxtJaGkk7Pf/cMKU54QEk++01oLNxuJ2C2qLChp1G2alrUxHaPIbX4LKcSSafrbeCsnwBeRJVgj6qqvMUWOCCLn07c="
```

**Comment by pedrobaeza** [comment]
I needed to remove the virtualenv, which is not valid for Python 3. Now we have a green build, so I fast-track this for having CI tools in the repo. I squash to not fill too much commits for this tooling stuff.

**Pull request merged**